boom/mounts.py

```
@@ -86,6 +80,13 @@ def _parse_mount_unit(mount):
     else:
         options = "defaults"
 
+    if not what.startswith(_DEV_PREFIXES):
+        raise BoomMountError(f"Invalid mount device: {what}")
+
+    if not where.startswith("/"):
+        if where != "none" and fstype != "swap":
+            raise BoomMountError(f"Invalid mount point: {where}")
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)